examples/06-custom-schema/06-togglable-blocks/README.md

```diff
@@ -0,0 +1,8 @@
+# Togglable Blocks
+
+This example shows how to create blocks with a toggle button to show/hide their children, like with the default toggle heading and list item blocks. This is done using the use the `ToggleWrapper` component from `@blocknote/react`.
```

Comment:
let's make explicit (both in the example name and description) that this is about custom blocks